### DISCUSSION TITLE
How to increase the display scaling with WSL2?

### DISCUSSION THREAD

**Discussion by ArchieCoder** [unoplatform/uno#16736] [General]
When I debug my app with WSL2, my app renders with small text.

I saw this information on Uno doc, but I don't know where to call FrameBufferHost.DisplayScale.
![image](https://github.com/unoplatform/uno/assets/1608424/dc5d26f1-b8be-4465-8336-9ed8197fc8dd)

**Comment by jeromelaban** [comment]
Let's track in https://github.com/unoplatform/uno/issues/16742
